### PR TITLE
{bio} [dummy] Coot (REVIEW)

### PR DESCRIPTION
**Pull request by a-mazur**
Fixes permissions for group and others

### CODE REVIEW

File: easybuild/easyconfigs/c/Coot/Coot-0.8.1-binary-Linux-x86_64-rhel-6-python-gtk2.eb [hunk 1]
```diff
@@ -18,6 +18,9 @@ toolchain = {'name': 'dummy', 'version': ''}
 source_urls = ['http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot/binaries/release']
 sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
 
+postinstallcmds = ['chmod -R a=u %(installdir)s',
+                   'chmod -R g-w,o-w %(installdir)s']
```
Comment (boegel) [inline easybuild/easyconfigs/c/Coot/Coot-0.8.1-binary-Linux-x86_64-rhel-6-python-gtk2.eb]: can you reformat this slighty?

```python
postinstallcmds = [
    'chmod -R a=u %(installdir)s',
    'chmod -R g-w,o-w %(installdir)s',
]
```

and maybe also add a comment (in the easyconfig) what you're fixing exactly?

Also, do you need to use two separate commands?

Comment (fgeorgatos) [inline easybuild/easyconfigs/c/Coot/Coot-0.8.1-binary-Linux-x86_64-rhel-6-python-gtk2.eb]: @boegel: what's your take for an one-liner here?

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/c/Coot/Coot-0.8.1-binary-Linux-x86_64-rhel-6-python-gtk2.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/c/Coot/Coot-0.8.1-binary-Linux-x86_64-rhel-6-python-gtk2.eb]: I'd go with the above, to make it clear that there are 2 commands.

**Comment by migueldiascosta** [comment]
@a-mazur, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**